operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -894,6 +894,16 @@ function wdtCreatePrimordialDomain() {
 
   trace "About to call '${WDT_BINDIR}/createDomain.sh ${wdtArgs}'."
 
+  trace "DEBUG: WLSDEPLOY_PROPERTIES is ${WLSDEPLOY_PROPERTIES}"
```

Comment:
There is dump of env values when the introspect job starts,  I am wondering should we revisit the list we dump and add it there (and possibly others) instead?

Reply:
I added WLSDEPLOY_PROPERTIES to the list. From the comment in traceEnv function, we purposely avoid dumping all env vars. Not sure what else needs to be added to this list.

Reply:
Thanks.  Can you check with Lenny or Tom ?  I think Lenny looked at an issue before and wondering what the value of a variable.  It's ok if no one recall any.